frontend/src/components/WofSmallError.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+<div class="small-error">
+    <div class="small-error__contener">
+        <div class="small-error__icon">
+            <wof-icon @mouseover="hover = true" @mouseleave="hover = false" icon="wof-warning-circle-fill" :size="this.size"/>
+        </div>
+        <div class="small-error__wrap-text">
+            <div class="small-error__content" v-if="hover">
+                <slot></slot>
```

Comment:
I think we should use "text" prop instead of slot because this component is for text errors.